Docker/build/WikibaseBundle/LocalSettings.d.template/WikibaseCirrusSearch.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
 // https://www.mediawiki.org/wiki/Extension:WikibaseCirrusSearch
+wfLoadExtension( 'WikibaseCirrusSearch' );
```

Comment:
similar to above

Reply:
Same comment as above (and as written in the reverting commit msg)
This is just undoing https://phabricator.wikimedia.org/T299393 so that wmde.5 and wmde.6 both do not have this fix.
Then both of the next releases (whenever they would come, an also hopfully once this glue has release notes too) would have the fix